ci/functional/required_packages.sh

```diff
@@ -32,7 +32,9 @@ elif [[ $distro = el* ]] || [[ $distro = centos* ]] ||
           hdf5-vol-daos-mpich-tests    \
           MACSio-mpich                 \
           MACSio-$openmpi              \
-          mpifileutils-mpich"
+          mpifileutils-mpich           \
+          daos-server-tests-openmpi    \
+          daos-client-tests-openmpi"
```

Comment:
Use of this list of dependencies and this file entirely is being deprecated.  It needs to go away completely at some point.  Dependency information belongs in RPMs.  The existence of this file is papering over lack-of-dependency bugs.

Reply:
Fixed